packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -1083,12 +1083,17 @@
     {{! template-lint-disable no-inline-styles }}
     <ul class="hds-dropdown-list" style="width: 250px">
       {{#each @model.ITEM_STATES as |state|}}
-        <Hds::Dropdown::ListItem::CopyItem @text="{{state}}: 91ee1e8ef65b337f0e70d793f456c71d" @state={{state}} />
+        <Hds::Dropdown::ListItem::CopyItem
+          @text="{{state}}: 91ee1e8ef65b337f0e70d793f456c71d"
+          mock-state-value={{state}}
+          mock-state-selector="button"
```

Comment:
here instead you can see a case where the _target_ element is **inside** the component, but using the `mock-state-selector` it's possible to apply the mock state class directly to it, without the need to pass down arguments across components and sub-components